app/uk/gov/hmrc/agentpermissions/controllers/TaxServiceGroupsController.scala

```diff
@@ -192,6 +192,22 @@ class TaxServiceGroupsController @Inject() (taxGroupsService: TaxGroupsService)(
     }
   }
 
+  def removeTeamMember(gid: String, memberId: String): Action[AnyContent] = Action.async { implicit request =>
+    withAuthorisedAgent() { authorisedAgent =>
+      taxGroupsService
+        .removeTeamMember(gid, memberId, authorisedAgent.agentUser)
+        .map {
+          case AccessGroupNotUpdated =>
+            logger.info(s"Custom group '$gid' didn't remove client '$memberId''")
+            NotModified
+          //          case AccessGroupUpdated => NoContent
+          case AccessGroupUpdatedWithoutAssignmentsPushed =>
+            logger.info(s"Custom group removed a team member, but assignments were not pushed")
```

Comment:
These should have different cases. No need to mention assignments for TSG
```suggestion
          case TaxServiceGroupNotUpdated =>
            logger.info(s"Tax service group '$gid' didn't remove team member '$memberId''")
            NotModified
    
          case TaxServiceGroupUpdated =>
            
```